activemodel/lib/active_model/type/big_integer.rb

```diff
@@ -4,7 +4,21 @@
 
 module ActiveModel
   module Type
-    class BigInteger < Integer # :nodoc:
+    # Attribute type for integer representation with no byte range limit for
+    # serialization. This type is registered under the +:big_integer+ key.
```

Comment:
To match the terminology we used in `ActiveModel::Type::Integer`:

```suggestion
    # Attribute type for integers that can be serialized to an unlimited number 
    # of bytes. This type is registered under the +:big_integer+ key.
```